Intersect.Server/Entities/Npc.cs

```diff
@@ -1633,6 +1633,24 @@ public override EntityPacket EntityPacket(EntityPacket packet = null, Player for
             return pkt;
         }
 
+        public override void OnNearEntityDies(Entity en)
+        {
+            if (en.GetEntityType() == EntityType.Player)
+            {
+                RemoveFromDamageMap(en);
+            }
+        }
+
+        public override void OnNearEntityAttacked(Entity victim, Entity attacker) 
+        {
+            if (Target == null &&
+                IsAllyOf(victim) &&
+                InRangeOf(this, Base.SightRange) &&
+                victim.GetEntityType() == EntityType.Player)
```

Comment:
Also to avoid line breaks in the middle of an if statement, invert the conditions and return